sips/fork_support.md

```diff
@@ -161,7 +160,7 @@ func (networkID NetworkID) ForkAtEpoch(epoch phase0.Epoch) (*ForkData, error) {
 
 	// If empty, raise error
 	if len(forks) == 0 {
-		return nil, errors.New("GetCurrentFork: fork list by GetForksData is empty.")
+		return nil, errors.New("GetCurrentFork: fork list by GetForksData is empty. Unknown Network")
```

Comment:
let's keep error format to what we use currently. "GetCurrentFork:..." is not how we write errors

Reply:
```suggestion
		return nil, errors.New("Fork list by GetForksData is empty. Unknown Network")
```